conf/layer.conf

```diff
@@ -8,7 +8,22 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "celebrity-face-match"
 BBFILE_PATTERN_celebrity-face-match = "^${LAYERDIR}/"
 BBFILE_PRIORITY_celebrity-face-match = "24"
-LAYERSERIES_COMPAT_celebrity-face-match = "hardknott"
-LAYERDEPENDS_celebrity-face-match = "phytec ampliphy meta-ml"
+LAYERSERIES_COMPAT_celebrity-face-match = "scarthgap"
+LAYERDEPENDS_celebrity-face-match = "phytec ampliphy imx-machine-learning"
 
-DISTRO_VERSION_append = "-CelebrityFaceMatch-v0.6"
+BBMASK += "meta-ampliphy/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-imx/meta-bsp/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-freescale-distro/recipes-graphics/wayland/weston-init.bbappend"
```

Comment:
This and the weston.ini itself are the reason why the kirkstone PR #5 has not yet been merged: this is necessary to run on LVDS display, yet the AI kit does not come with such a display, so the default should be the HDMI ouput.

Reply:
Understood. Will change the default to HDMI then.

Reply:
I have removed the corresponding commits and used the default coming from meta-ampliphy and other layers. This should enable HDMI by default.

Reply:
That is the case. Yet just leaving it as in meta-ampliphy lets the way how to switch to LVDS undocumented or better: one has to actively search for the weston.ini part. That was the open part that I was still undecided about in #5: keep it as a reminder/easyswitch for setting up our fair demo or completely remove it.

Reply:
Just had a look into that issue: as noted in #5, there is no reason to keep the weston.ini in any way. I somehow thought that there was more about that... Removed it also on the kirkstone PR.